src/main/java/com/andrewthesailor/cinemaTickets/model/Jsons/TicketList.java

```diff
@@ -0,0 +1,6 @@
+package com.andrewthesailor.cinemaTickets.model.Jsons;
+
+import java.util.ArrayList;
+
+public class TicketList extends ArrayList<TicketData> {
```

Comment:
A  co to za cudak?

Reply:
W praktyce niepotrzebny, umknął mi jak robiłem przegląd. Trochę nie jestem od kilku dni w stanie nadającym się do myślenia.